src/Logger.ts

```diff
@@ -14,47 +14,74 @@ export class Logger implements LoggerService {
     this.contextName = renameContext || "context";
   }
 
-  verbose(message: any, context?: string, ...args: any[]) {
+  verbose(message: any, context?: string, isTimestampEnabled?: boolean, ...args: any[]) {
```

Comment:
obviously that will be breaking change and actually make usage of this logger not convenient